docs/csharp/language-reference/builtin-types/snippets/StructType.cs

```diff
@@ -95,4 +95,22 @@ public int Counter
             // </SnippetReadonlyOverride>
         }
     }
+
+    namespace ref_struct
+    {
+        // <SnippetRefStruct>
+        public ref struct CustomRef
+        {
+            public Span<int> Inputs;
+            public Span<int> Outputs;
+        }
+        // </SnippetRefStruct>
+
+        // <SnippetReadonlyRef>
+        public readonly ref struct Example
+        {
+            // Type member definitions
```

Comment:
Consider adding an integer, or a boolean here to show examples of those types in a ref struct.

Reply:
@BillWagner should I also add a span?

In general, is it worth to create custom structs **without** `ref` struct fields? I've thought that the main reason to define a custom `ref` struct is to encapsulate already existing ones.

Reply:
> In general, is it worth to create custom structs without ref struct fields? I've thought that the main reason to define a custom ref struct is to encapsulate already existing ones.

I think most scenarios outside of corefx will be combining existing `ref struct` types. That said, many `ref struct` and `readonly ref struct` types in the corefx library contain a mix of other `ref struct` types and primitive unmanaged types as members. It's worth mentioning that it is legal.

Reply:
@BillWagner I've updated both sample examples to include some simple types. Do you think that would be enough?